src/tests/index.js

```diff
@@ -1,65 +1,52 @@
-const LoginPage = require('../pages/login.page.js');
-const SearchPage = require('../pages/search.page.js')
-let basePage = new BasePage()
-let loginPage = new LoginPage()
-let searchPage = new SearchPage()
+const BasePage = require("../pages/base.page.js");
+const LoginPage = require("../pages/login.page.js");
+const SearchPage = require("../pages/search.page.js");
+let basePage = new BasePage();
+let loginPage = new LoginPage();
+let searchPage = new SearchPage();
 
 //let driver = await new Builder().forBrowser('chrome').build();;
-describe('First script', function () {
+describe("First script", function () {
+  //авторизация
+  it("Avtorization", async function () {
+    await basePage.openPage("https://www.saucedemo.com/");
 
-      //авторизация
-      it('Avtorization', async function () {
+    const currentURL = await basePage.getPageUrl();
+    console.log(currentURL.toString());
 
-       await basePage.openPage('https://www.saucedemo.com/')
-       
-        const currentURL = await basePage.getPageUrl();
-        console.log(currentURL.toString());
-   
-    
-        await loginPage.loginUser('standard_user', 'secret_sauce')
-        const newURL = await basePage.getPageUrl();
+    await loginPage.loginUser("standard_user", "secret_sauce");
+    const newURL = await basePage.getPageUrl();
 
-        expect(newURL).equal('https://www.saucedemo.com/inventory.html');
-        
-      
-       // await new Promise(resolve => setTimeout(resolve, 5000));
-      });
+    expect(newURL).equal("https://www.saucedemo.com/inventory.html");
 
-       //   //AddtoCard
-     
-      it('AddtoCard', async function () {
-    
-      await searchPage.addToCard()
-      const checkBadgeElement = await basePage.findElement(searchPage.badge)
-      const checkBadgeValue = await basePage.getElementAttribute(checkBadgeElement)
-      console.log(checkBadgeElement)
-      expect(checkBadgeValue).to.equal('1')
-    } );
-    
-   
-     
+    // await new Promise(resolve => setTimeout(resolve, 5000));
+  });
 
-    
- 
+  //   //AddtoCard
+
+  it("AddtoCard", async function () {
+    await searchPage.addItemToCart();
+   // const checkBadgeElement = await basePage.findElement(searchPage.badge);
+    const checkBadgeValue =
+      await basePage.getElementAttribute(searchPage.badge); // (searchPage.badge)
+    console.log(checkBadgeElement);
+    expect(checkBadgeValue).to.equal("1");
+  });
 
   //   //Remove
 
-  it('Remove from cart', async function () {
+  it("Remove from cart", async function () {
+    await searchPage.removeItemFromCart();
 
-  await searchPage.removeFromBadge()
-  
-  const isBadgeDisplayed = await basePage.isElementDisplayed(searchPage.badge)
-  expect(isBadgeDisplayed).to.equal(false)
+    const isBadgeDisplayed = await basePage.isElementDisplayed(
+      searchPage.badge,
+    );
+    expect(isBadgeDisplayed).to.equal(false);
+  });
 
+  after(async () => await basePage.quitPage());
 });
```

Comment:
Add tests for Product page
1 test: check that Product page is displayed (check url)
Check item name is displayed

2 test: Add to cart from Product page
Check that item on cart is displaying 1 item